guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -35,7 +35,7 @@ Use the following layout instructions to customize your layout:
 *  <a href="#argument"><code>&lt;argument&gt;</code></a>
 *  <a href="#arguments"><code>&lt;arguments&gt;</code></a>
 
-### &lt;block&gt; {#fedg_layout_xml-instruc_ex_block}
```

Comment:
I would not eliminate the "<" and the ">" in this title, it references a node inside layout.